testgrid/config/config.yaml

```diff
@@ -16,6 +16,7 @@ default_test_group:
   column_header:
   - configuration_value: Commit # Shows the commit number on column header
   - configuration_value: infra-commit
+  num_columns_recent: 10
```

Comment:
Nope; it'll mean that for logic that cares about a number of recent columns (for instance, how many columns back to look to see if a tab is flaky), we'll be able to calculate that.